### DISCUSSION TITLE
Remove unnecessary localize resource DLLs from `docfx` nuget package

### DISCUSSION THREAD

**Discussion by filzrev** [dotnet/docfx#8813] [Ideas]
When execute `dotnet build` command to  `docfx` project.
Following satellite DLL folders are created to output directory. 

- cs
- de
- es
- fr
- it
- ja
- ko
- pl
- pt-BR
- ru
- tr
- zh-Hans
- zh-Hant

There directories are introduced by `Microsoft.CodeAnalysis.*` NuGet packages.
(`Microsoft.DocAsCode.Dotnet` project reference these packages)
And it takes about `11MB` storage size per target frameworks (`net6.0`/`net7.0` and `net8.0`(for CI build))

---
Currently `docfx.exe` itself don't have localized text resources.
So it may be better to suppress these output.

---
I've confirmed, these outputs can be suppressed  by adding following settings to `docfx.csproj`.

```xml
<PropertyGroup>
  <SatelliteResourceLanguages>en</SatelliteResourceLanguages>
</PropertyGroup>
```

On my local environment. `.nupkg`  (created by `dotnet pack` ) size is reduced from `35MB` to `28MB`.  
(`.nupkg` is zip archived, so duplicated contents don't affects total archive size)

On latest `docfx.2.67.2.nupkg` ,  file size will be reduce about `84MB`->`74MB`

---
I hope this idea will be accepted. (If requested, I'll create new PR)

**Comment by yufeih** [comment]
> Currently docfx.exe itself don't have localized text resources.

That makes sense. Docfx itself isn't localized, it could be used to build multilingual websites though. It appears localized `Microsoft.CodeAnalysis.*` packages won't affect generated build output, except for log messages, so the idea should work. It's a trivial change I can do.

**Reply by yufeih** [reply]
Done in https://github.com/dotnet/docfx/commit/6d91d6c0931963cf91e894e6f4b8f80cbdb249b5
